tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -275,6 +275,7 @@ class AirloopUnitarySysTest : public EnergyPlusFixture
         state->dataSize->SysSizPeakDDNum(1).CoolFlowPeakDD = 1;
         state->dataSize->SysSizPeakDDNum(1).TotCoolPeakDD = 1;
         state->dataSize->FinalSysSizing.allocate(1);
+        state->dataSize->CalcSysSizing.allocate(1);
```

Comment:
New references in function GetCoilDesFlowT now requires the allocation of CalcSysSizing in unit test fixtures.